### DISCUSSION TITLE
What would be a great community to focus on?

### DISCUSSION THREAD

**Discussion by davidgasquez** [datonic/hub#26] [Ideas]
On top of my mind, there are a few communities that will benefit from having an awesome open data landscape.

- Blockchains.
- Open Gobernance projects like Gitcoin, Open Source Observer, ...
- Decentralized Science. 

One of the best things to do now would be to find a community where people care about this and start providing useful datasets!

A few reasons:

* Most data is super messy and it not accessible through nice APIs. Technical skills are needed to process and ingest it.You might need custom ingestion for each niche and it's pretty likely you'll need some rules to standardize data from multiple sources, neither of which seems easy to generalize and automate because they're very domain-specific.
* You get easier distribution. If the data is useful the community will share it within itself and spread the word.
* You're closer to the problem than curating another dataset covering economic indicators for countries.
* You don't have to compete with HF, Google, and the likes.
* There is much more low hanging fruit in small to medium communities.
* With LLMs on the rise, community curated datasets become more important as they don't appear in the big data dumps. Frontline communities around the world suffer from a critical data gap that holds them back from making efficient decisions.
* Much cheaper to run. No big data (probably)!

**Comment by davidgasquez** [comment]
Landing somewhere around decentralized communities. Datonic could cover a few interesting gaps:

- Data Portals for communities like Gitcoin, Arbitrum, ...
- Data Packages and Data Pipelines for DeSci.
- Analytitcs / Reporting for DAOs, Social Media (Farcaster, Bluesky, ...)

**Reply by davidgasquez** [reply]
For the last year or so, I've been maintaining the following "Datadex" projects:

- [Gitcoin Grants Data Portal](https://github.com/davidgasquez/gitcoin-grants-data-portal)
- [Filecoin Data Portal](https://github.com/davidgasquez/filecoin-data-portal/)

Both have received funding from their respective ecosystems by [retroactive public goods funding](https://medium.com/ethereum-optimism/retroactive-public-goods-funding-33c9b7d00f0c).

**Comment by TomNicholas** [comment]
https://www.pangeo.io/ :)

We have:
- Societally-important and commercially-valuable datasets
- Small numbers of high-impact and multi-user datasets, with a long tail of lesser-used datasets
- Massive datasets requiring high throughput (so if your solution works for us it will work at any scale)
- Data model and stack that generalizes to most scientific datasets (anything that's multidimensional arrays)
- Active and forward-looking open-source software community
- Interest in decentralized and content-addressed ideas
